### ISSUE TITLE
please register it in the ore dictionary

### ISSUE THREAD

**Issue by RCXcrafter** [diesieben07/Modjam-4#7]
None of the ores/ingots/gems/blocks are registered in the ore dictionary, this is required for mods like stacksonstacks or AOBD to work properly.

**Comment by diesieben07** [comment]
- This has nothing to do with the OreDictionary.
- I might add support for StacksOnStacks, that looks fun.
- I might be stupid, but I do not get the point of AOBD. Does it just add dusts for every ore it finds? Then no thank you I'd rather not some mod just create new Items for my mod.

**Comment by RCXcrafter** [comment]
What do you mean with "This has nothing to do with the OreDictionary."? If you make a mod with ores it's always a good idea to use the ore dictionary so that your copper can be used with copper from other mods.

StacksOnStacks also works with the ore dictionary, it automatically makes every ingot/gem/dust placable if it's registered in the ore dictionary.

AOBD only adds dust if there is a mod detected that can grind ores into dust, it also adds other items for machines from other mods so that every machine from the supported mods (quite a lot) can process every ore from every mod.

So please consider adding your items to the ore dictionary.

**Comment by diesieben07** [comment]
> If you make a mod with ores it's always a good idea to use the ore dictionary
No, not necessarily. The OreDictionary exists so that (as you say) e.g. two different copper Items are compatible. Show me a mod that adds any of the Items in BadOres and I'll add them to the OreDict.

About StacksOnStacks: Yes, it has the OreDictionary as _one_ source. But it has an API which is the actual way to make your mod interoperate with it.

About AOBD: That is what I thought. So no, I do not want this mod to just go ahead and add dusts for the ores. It makes no sense for them to exist.

**Comment by RCXcrafter** [comment]
Well, your copper items aren't in the ore dictionary either.

AOBD only adds the dust if there is an ore and an ingot found for a certain material so that ores can be processed by machines that can multiply your ores, which can actually be useful if you want to use nopium for decoration (it looks pretty good as a deco block).

here is another mod that requires items to be registered in the ore dictionary: http://minecraft.curseforge.com/projects/colorblindhelper

**Comment by diesieben07** [comment]
> Well, your copper items aren't in the ore dictionary either.

I will address this.

IMHO opinion any mod that relies on the OreDict to detect any kind of item is stupid. That's not the purpose of the OreDict.
But since as always mod makers are stupid and seem to abuse it in this way, I will have to budge and comply and abuse it as well.
Oh well.

**Comment by RCXcrafter** [comment]
How could this use of the ore dictionary cause anything bad to happen?

**Comment by diesieben07** [comment]
I did not say it would.
It is just using the wrong tool for the job.
